Bottom: 06a2d73be01052cdca421a8f49ae4ec88d7043ea
Top:    1841c30b5d859069adb077d5c5f28a6dbabdfb6c
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-05 23:07:03 +0100

Refresh of hid-gtk-connect-up-the-file-ch

---

diff --git a/src/hid.h b/src/hid.h
index b1f532f..afb33e7 100644
--- a/src/hid.h
+++ b/src/hid.h
@@ -567,6 +567,16 @@ typedef enum
      * Any remaining rendering will be flushed to the screen.
      */
     void (*finish_debug_draw)  (void);
+
+    /* Notification to the GUI around saving the PCB file.
+     *
+     * Called with a false parameter before the save, called again
+     * with true after the save.
+     *
+     * Allows GUIs which watch for file-changes on disk to ignore
+     * our deliberate changes.
+     */
+    void (*notify_save_pcb) (bool done);
   };
 
 /* Call this as soon as possible from main().  No other HID calls are
